src/ProjectOrigin.WalletSystem.Server/Database/Scripts/202308041300-number-increment.sql

```diff
@@ -0,0 +1,26 @@
+CREATE TABLE IdIncrements (
+    Id uuid PRIMARY KEY,
+    number INT NOT NULL
+);
+
+CREATE OR REPLACE FUNCTION IncrementNumberForId(
```

Comment:
Hvorfor denne og ikke bare WalletPosition +1?

Reply:
I created a new table and stored procedure to ensure unique increments for any given uuid.